billing-db/tables/resources.sql

```diff
@@ -19,5 +19,5 @@ CREATE TABLE IF NOT EXISTS resources
 	last_updated TIMESTAMPTZ NOT NULL -- When this row was last updated
 );
 
-CREATE INDEX CONCURRENTLY resources_i1 ON resources (valid_from, valid_to);
-CREATE INDEX CONCURRENTLY resources_i2 ON resources (org_name, valid_from, valid_to);
+CREATE INDEX IF NOT EXISTS resources_i1 ON resources (valid_from, valid_to);
```

Comment:
You could have `CREATE INDEX CONCURRENTLY IF NOT EXISTS resources_i1...` here, just in case this is being run against an already-existing instance of `resources`

Reply:
Looking back at the commit history of the new-billing-integration it looks like the concurrently key words were removed due to them not being allowed during a transaction. @schmie can you give more context?

Reply:
The golang execution within a transaction crashes at CONCURRENTLY.